internal/test/wsu/wsu_test.go

```diff
@@ -44,6 +39,38 @@ var (
 	ubi8Image = "registry.access.redhat.com/ubi8/ubi:latest"
 )
 
+type WsuFramework struct {
+	framework *e2ef.TestFramework
```

Comment:
You can just embed the framework here. There is no need to name it as `framework`. That will make the referencing of its fields in later code more succinct.